package.json

```diff
@@ -6,20 +6,26 @@
         "type": "git",
         "url": "https://github.com/NordicSemiconductor/pc-nrfconnect-launcher.git"
     },
-    "main": "src/main",
+    "main": "dist/main.js",
     "scripts": {
         "postinstall": "node getJlink.js",
-        "app": "electron .",
-        "dev": "webpack watch --mode development",
-        "webpack": "webpack build --mode development",
-        "build": "webpack build",
-        "test": "jest src/",
+        "app": "NODE_OPTIONS=--enable-source-maps electron .",
```

Comment:
Doesn't work well in windows unfortunately

Reply:
Yes, I totally forgot that this is doesn't work on Windows. But luckily it is easily with [cross-env](https://github.com/kentcdodds/cross-env), which I will do.